run/idp-sql/requirements.txt

```
@@ -1,5 +1,5 @@
 Flask==3.0.3
-SQLAlchemy==2.0.24
+SQLAlchemy==2.0.36
```

Larger broken issue with this sample, remove from this large batch update

```suggestion
SQLAlchemy==2.0.24
```